cmd/pool/order.go

```diff
@@ -257,7 +263,14 @@ func ordersSubmitAsk(ctx *cli.Context) error { // nolint: dupl
 
 	ask := &poolrpc.Ask{
 		LeaseDurationBlocks: uint32(ctx.Uint64("lease_duration_blocks")),
-		Version:             uint32(order.VersionLeaseDurationBuckets),
+		Version:             uint32(order.VersionSidecarChannel),
+	}
+
+	// Because there is no disadvantage of being matched against bids with
+	// push amounts (other than perhaps a bit more complicated accounting
+	// formulas), we make them opt-out.
+	if ctx.Bool("disallow_push_amt") {
```

Comment:
hm, is there really any compelling reason that we know of to make this optional?

Reply:
IMO a maker shouldn't really care about this, given that with the way it ends up being executed, their funds aren't used for the push amount itself. However maybe there's soem routing related reason they wouldn't want to have a balanced channel in the first place? Not sure...

Reply:
My concern here was mainly the accounting of the maker. Their Pool account balance wouldn't decrease by the expected amount (`capacity - premium`) and instead some remote channel balance would suddenly appear in their `lnd` balance. I'll look at this again in the execution PR. Maybe it's enough if we just correctly set the `push_amt` in the matched order so the maker knows what exactly went down. Going to remove the commit for now, easy to add again later on.

Reply:
This is solved with the self chan balance now being additive. Therefore the maker's account balance doesn't change at all compared to orders without a push amount.